site/app/controllers/admin/ConfigurationController.php

```diff
@@ -112,6 +153,20 @@ public function updateConfiguration() {
             $entry = nl2br($entry);
         }
 
+        if($name === 'forum_enabled') {
+            if($entry == 1){
+                if($this->core->getAccess()->canI("forum.modify_category")) {
+                    $categories = ["General Questions 2", "Homework Help 2", "Quizzes 2" , "Tests 2"];
```

Comment:
Is there a reason that these have a trailing 2 (`*CategoryName* 2`)? I'm not sure if I missed a conversation but should it just be `General Questions`, `Homework Help`, `Quizzes`, `Tests`?

Reply:
Must have crept when I was testing the removal of ForumController, will get that fixed!